assemble

```diff
@@ -43,6 +43,8 @@ pushd /tmp/src
   else
       thamos advise --no-wait
   fi
+} || {
+  echo "Thoth recommendations are not activated"
```

Comment:
do we call it recommendation or advise? :)

Reply:
Let's call it advise :+1: